### PR TITLE
Add schema tests for 5490

### PR DESCRIPTION
**Pull request by jbalboni**
Related to [1612](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1612)

Adds schema tests for 5490, following what we did for the 1995. There are some schema updates, too, which I found in creating the tests.

### CODE REVIEW

File: test/edu-benefits/5490/schema/minimal-test.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
+  "privacyAgreementAccepted": true,
+  "submission": {
```
Comment (raquelromano-gov) [inline test/edu-benefits/5490/schema/minimal-test.json]: Why does this file have a "submission" field but the maximal example doesn't? Trying to follow suit for the 1990e schema tests in https://github.com/department-of-veterans-affairs/vets-website/tree/1990e-schema-test-rr

Comment (jbalboni) [inline test/edu-benefits/5490/schema/minimal-test.json]: I just forgot to remove in here. It's not necessary.

**Comment by raquelromano-gov** [comment]
LGTM

File: test/edu-benefits/5490/schema/highschool-test.json [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+          "dateRange": {
+            "from": "2011-11-01",
+            "to": "2010-12-12"
+          },
+          "name": "Test high school",
+          "city": "Northampton",
+          "state": "MA",
+          "hours": 23,
+          "hoursType": "semester",
+					"degreeReceived": "Diploma"
```
Comment (cvalarida) [inline test/edu-benefits/5490/schema/highschool-test.json]: Weird indentation.

File: test/common/schemaform/helpers.unit.spec.js [hunk 3]
```diff
@@ -741,6 +741,33 @@ describe('Schemaform helpers:', () => {
...
+            pages: {
+              page1: {}
+            }
+          }
+        }
+      };
+      const formData = {
+        page1: {
+          data: {
+            someField: {
```
Comment (raquelromano-gov) [inline test/common/schemaform/helpers.unit.spec.js]: If the form config is empty, does it matter whether these fields are empty, undefined, or defined? Just wondering what scenario this is testing as opposed to
```
formData = {};
```
or 
```
formData = {
  page1: {
    data: {}
  }
};

File: test/edu-benefits/5490/schema/maximal-test.json [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+          },
+          "serviceStatus": "Great"
+        }
+      ]
+    }
+  },
+  "contributions": {
+    "data": {
+      "civilianBenefitsAssistance": true,
+      "civilianBenefitsSource": "The govt?"
```
Comment (cvalarida) [inline test/edu-benefits/5490/schema/maximal-test.json]: Ha! Brilliant!

**Review by cvalarida — COMMENTED** [review]
Looks good. Lots of data.

File: test/common/schemaform/helpers.unit.spec.js (hunk 3 above)
Comment (jbalboni) [inline test/common/schemaform/helpers.unit.spec.js]: There will always be a data object underneath each page's object which has the form data. But inside that it's partly determined by how you filled out the form whether there are empty objects or undefined fields in spots. 

There's a change in this PR to replace empty objects with undefined before sending to the server, so it shouldn't matter in practice, though we could update ObjectField to clean up empty objects, too.

File: test/edu-benefits/5490/schema/minimal-test.json [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  "previousBenefits": {
+    "data": {
+      "previousBenefits": {
+        "veteranFullName": {}
+      }
+    }
+  },
+  "sponsorVeteran": {
+    "data": {
+      "relativeFullName": {
```
Comment (raquelromano-gov) [inline test/edu-benefits/5490/schema/minimal-test.json]: Just noticed this--shouldn't this be veteranFullName?

Comment (cvalarida) [inline test/edu-benefits/5490/schema/minimal-test.json]: Ah, I think you're right, @raquelromano-gov. It's currently `relativeFullName` in the config right now, so I'll fix that up when I'm making the tweaks to the 5490.

**Pull request merged**